src/gbt_convertors.pyx

```diff
@@ -242,3 +243,242 @@ def get_gbt_model_from_xgboost(booster: Any) -> Any:
             node_queue.append(Node(sub_tree["children"][1], parent_id, 1))
 
     return mb.model()
+
+def get_gbt_model_from_catboost(model: Any) -> Any:
+    if not model.is_fitted():
+        raise RuntimeError(
+            "Model should be fitted before exporting to daal4py.")
+
+    dump_filename = str(time()) + '_' + str(getpid())

```

Comment:
I'd have probably added a short description of a file
Something like: `f'catboost_model_{getpid()}_{time()}'`
I also think that process ID would better stand before time - that way it would be easier to sort such files if needed